[Workflow] Allow use of UnitEnum and BackedEnum in workflow places (#44306)
symfony/symfony: src/Symfony/Component/Workflow/CHANGELOG.md

CHANGELOG
=========

6.1
---

>  * Added support of UnitEnum and BackedEnum to Workflow places

Reviewer: ```suggestion
 * Allow use of UnitEnum and BackedEnum in workflow places
```
